### PR TITLE
OP-1235 Reflecting changes in CORE

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1235.

Paired with https://github.com/informatici/openhospital-core/pull/1326

Also fix naming test helper class conflicting with core

### CODE REVIEW

File: src/test/java/org/isf/admission/gui/ward/WardBrowserManagerStub.java [hunk 1]
```diff
@@ -24,11 +24,22 @@
...
+import org.mockito.Mock;
 
 public class WardBrowserManagerStub extends WardBrowserManager {
 
+	@Mock
+	private WardIoOperations wardIoOperations;
+
+	public WardBrowserManagerStub(AdmissionBrowserManager admissionBrowserManager, WardIoOperations wardIoOperations) {
+		super(admissionBrowserManager, wardIoOperations);
+		// TODO Auto-generated constructor stub
```
Comment (dbmalkovsky) [inline src/test/java/org/isf/admission/gui/ward/WardBrowserManagerStub.java:40]: remove this comment?

**Review by dbmalkovsky — APPROVED** [review]
LGTM with one small comment.

File: src/test/java/org/isf/admission/gui/ward/WardBrowserManagerStub.java (hunk 1 above)
Comment (mwithi) [inline src/test/java/org/isf/admission/gui/ward/WardBrowserManagerStub.java:40]:
```suggestion
```

Comment (mwithi) [reply src/test/java/org/isf/admission/gui/ward/WardBrowserManagerStub.java]: Done, thanks!

**Pull request merged**